docs/design/project-name.md

```diff
@@ -0,0 +1,32 @@
+# Enhancement Title
+
+> [Issue #428](https://github.com/redhat-et/issues/428)
+
+## Summary
+
+Propose a new name for this project: Nexodus.
+
+## Proposal
+
+The current name (Apex) is heavily used, including in a number of places in IT.
+A more unique name for the project will make it more uniquely recognizable, as
+well as more memorable.
+
+* Proposed Project Name: **Nexodus**
+* Proposed GitHub URL: <https://github.com/nexodus-io/nexodus>
+* Proposed Project Domain: `nexodus.io`
+* Proposed Binary Names
+  * `apexd` to `nexd`
+  * `apexctl` to `nexctl`
```

Comment:
@russellb are you planning to bring rest of the stuff from #441 in this proposal?

Reply:
yes i'm working on incporating it and will post an update